packages/@ember/canary-features/index.ts

```diff
@@ -52,10 +53,10 @@ export const FEATURES = assign(DEFAULT_FEATURES, ENV.FEATURES);
   @public
 */
 export function isEnabled(feature: string): boolean {
-  let featureValue = FEATURES[feature];
+  let value = FEATURES[feature];
 
-  if (featureValue === true || featureValue === false) {
-    return featureValue;
+  if (value === true || value === false) {
+    return value;
```

Comment:
I think I did this because at one point the linter complained that `featureValue` is already defined in the outer scope (the function on L67). Not sure why it was complaining then and why it stopped complaining now, but it seems good to address it anyway since it can indeed be a bit confusing.